model/riscv_insts_vext_vset.sail

```diff
@@ -48,6 +48,11 @@ function handle_illegal_vtype() = {
    */
   vtype.bits = 0b1 @ zeros(sizeof(xlen) - 1); /* set vtype.vill */
   vl = zeros();
+
+  if rv_enable_callbacks then {
+    csr_update_callback("vtype", vtype.bits);
+    csr_update_callback("vl", vl);
+  };
   print_reg("CSR vtype <- " ^ BitStr(vtype.bits));
```

Comment:
I guess you can remove these `print_reg()` calls too since the `csr_update_callback()` can do that?

Reply:
Yes, deleted this function.